src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,59 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.impl.response.ShoppingCartResponseMapper;
+import mate.academy.spring.model.dto.response.ShoppingCartResponseDto;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/shopping-carts")
+public class ShoppingCartController {
+    private ShoppingCartService shoppingCartService;
+    private UserService userService;
+    private MovieSessionService movieSessionService;
+    private ShoppingCartResponseMapper shoppingCartResponseMapper;
+
+    @PutMapping("/movie-sessions")
+    public String addSessionForUser(@RequestParam Long userId,
+                                    @RequestParam Long movieSessionId) {
+        shoppingCartService
+                .addSession(movieSessionService.get(movieSessionId), userService.get(userId));
+        return String.format("Movie session with id %d "
+                + "for user with id %d "
+                + "was successfully added to the shopping cart", movieSessionId, userId);
+    }

```

Comment:
You just need to return a DTO in your controller, beautiful responses for the user is the front-end developers' business)